app/lib/contentdm/importer.rb

```diff
@@ -29,10 +32,21 @@ def self.import(input_file, data_path, default_model)
 
     def import
       @records.each do |record|
-        work = process_record(record)
-        Contentdm::Log.new("Adding #{work.id} to collection: #{collection_name}", 'info')
+        begin
+          work = process_record(record)
+          Contentdm::Log.new("Adding #{work.id} to collection: #{collection_name[0]}", 'info')
+        rescue
```

Comment:
We're losing the error here.  Can we change it to print the original error to the logger?
```ruby
rescue => e
  Contentdm::Log.new("Problem importing record #{record}: ERROR: #{e}", 'error')
```